tests/sai_qualify/sai_infra.py

```diff
@@ -183,6 +183,7 @@ def run_case_from_ptf(duthost, dut_ip, ptfhost, test_case, test_interface_params
     test_para = None
     if request.config.option.enable_ptf_sai_test:
         cmds.append("echo \"export PLATFORM={}\" >> ~/.bashrc".format(get_sai_running_vendor_id(duthost)))
+        cmds.append("source  ~/.bashrc")
```

Comment:
we can use 'sed' to check if the current value meets the environment(platform) needs, and update as needed